src/app/client/src/app/modules/shared/components/page-section/page-section.component.ts

```diff
@@ -160,6 +162,14 @@ export class PageSectionComponent implements OnInit {
     if (visits.length > 0) {
       this.visits.emit(visits);
     }
+    const $next = $('.slick-slider').find('.slick-next');
```

Comment:
why have we used Jquery code can't we handle this using angular

Reply:
Removed jquery, used the ngx core function to detect the click